tasks/main.yml

```diff
@@ -1,11 +1,26 @@
 ---
+- name: Deleted active/inactive sessions for marked users 
+  command: "pkill -u {{ item.username }}"
+  with_items: "{{ users_deleted }}"
+  register: pkill_result
+  changed_when: "pkill_result.rc == 0"
+  failed_when: false
+  tags: ['users','configuration', 'users_delete']
+
 - name: Deleted user removal
   user:
     name: "{{ item.username }}"
     force: yes
     state: absent
   with_items: "{{ users_deleted }}"
-  tags: ['users','configuration']
+  tags: ['users','configuration', 'users_delete']
+
+- name: Delete home directory of marked users
+  file:
+    path: /home/{{ item.username }}
```

Comment:
Doesn't the `force: yes` in the task "Deleted user removal" remove the home directory

Reply:
`remove: yes` would. `force: yes` does even more:

> This option forces the removal of the user account, even if the user is still logged in. It also forces userdel to remove the user's home directory and mail spool, even if another user uses the same home directory or if the mail spool is not owned by the specified user. If USERGROUPS_ENAB is defined to yes in /etc/login.defs and if a group exists with the same name as the deleted user, then this group will be removed, even if it is still the primary group of another user.

I would probably prefer `remove: true` instead.
https://linux.die.net/man/8/userdel

Reply:
Okay, so update this PR and I'm fine to merge